chapters/operatorsandexpressions.tex

```diff
@@ -105,7 +105,8 @@ \section{Operator Precedence and Associativity}\label{operator-precedence-and-as
 
 The postfix array index and postfix access operators are not merely expression operators in the normal sense that \lstinline!a.b[1]! can be treated as \lstinline!a.(b[1])!.
 Instead, these operators need to be considered jointly to identify an entire \lstinline[language=grammar]!component-reference! (one of the alternative productions for \lstinline[language=grammar]!primary! in the grammar) which is the smallest unit that can be seen as an expression in itself.
-Postfix access can only be applied immediately to a \lstinline[language=grammar]!component-reference!; not even parentheses around the left operand are allowed.
+Postfix access can only immediately be applied to a \lstinline[language=grammar]!component-reference!.
```

Comment:
This contradicts the previous explanation that postfix access is part of the component reference.

Reply:
I find the previous two sentences (_The postfix array index…_ and _Instead these operators…) problematic, and suggest that we replace them with something like this:
```
The postfix array index and postfix access operators can both be part of a \lstinline[language=grammar]!component-reference! (one of the alternative productions for \lstinline[language=grammar]!primary! in the grammar) and be applied to general expressions when the left operand is parenthesized.
```

Reply:
I added that, and then realized that the preceding text needed an update as well - so I rewrote that.
I hope it makes more sense now.

Reply:
Even though I'm not exactly sure what changes to the _preceding test_ that correspond to this comment, I can't immediately spot any problem with the current formulations related to this.  Resolving.